packages/react-sdk/src/components/ElementBar/FontFamilyButton/FontFamilyButton.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2025 Nordeck IT + Consulting GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { MenuItem, Select } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { useFontFamily } from '../../../lib/text-formatting/useFontFamily';
+import { useActiveElements, useElements } from '../../../state';
+import { TextFontFamily } from '../../../state/crdt/documents/elements';
+
+const FONT_FAMILY = [
+  'inter',
+  'Abel',
+  'Actor',
+  'Adamina',
+  'Chewy',
+  'Gwendolyn',
+  'Pirata One',
+];
+
+export function FontFamilyButton() {
+  const { t } = useTranslation('neoboard');
+  const { fontFamily, setFontFamily } = useFontFamily();
+
+  const { activeElementIds } = useActiveElements();
+  const activeElements = useElements(activeElementIds);
+  const elements = Object.values(activeElements);
+
+  // If no 'shape' elements exist, do not render the button
+  if (elements.every((element) => element.type !== 'shape')) {
```

Comment:
We do currently check whether the font buttons should be displayed here https://github.com/nordeck/matrix-neoboard/blob/624c21ffdf31f3074122e244e4d99b3632771653/packages/react-sdk/src/components/ElementBar/ElementBar.tsx#L38

→ The check can be removed here